src/mozanalysis/utils.py

```diff
@@ -26,3 +27,19 @@ def dedupe_columns(columns):
         d[col._jc] = col
 
     return list(d.values())
+
+
+def all_(l):
+    """Return the logical AND of the input list l.
```

Comment:
Well... bitwise, for primitives, or elementwise for certain collections -- maybe just "combines the elements of the iterable `l` using the `&` operator"?

Do you mind using Google docstrings? https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Well, this is a question with surprising depth. I had a specific problem (the builtin `all` not playing nicely with spark dataframe columns) and wrote a one-liner to scratch that itch. But the one-liner is very general and could do many things other than scratch my original itch (the elements of the iterable don't need to be Columns). So I gave it a more general first line and demoted the reason for the function's existence (and probably the main reason it'll be used - otherwise people would just use the builtin) to the second line. Was this a mistake - should I have kept it front and center? "Return the logical AND of a list of `Column`s"?

"Combines the elements of the iterable `l` using the `&` operator" is a hair's breadth from "reduce over `l` using `&`"; i.e. it's just a restatement of the function's body and isn't particularly useful documentation IMO? Really people want to know whether this will solve their specific problem?

W.r.t. Google docstrings - are you asking for `Args` and `Returns` sections? I can do that - anything else my eye missed in the spot-the-difference?